steering_controllers_library/src/steering_odometry.cpp

```diff
@@ -52,7 +52,7 @@ bool SteeringOdometry::update_odometry(
   const double linear_velocity, const double angular, const double dt)
```

Comment:
```suggestion
  const double linear_velocity, const double angular_velocity, const double dt)
```

Shall we take this as an opportunity to rename the function variable to angular_velocity to maintain consistency?

@christophfroehlich what do you think?

Reply:
https://github.com/ros-controls/ros2_controllers/pull/1149

Reply:
Awesome. Thank you